### DISCUSSION TITLE
SpatialTC ERROR

### DISCUSSION THREAD

**Discussion by VladimirSobral** [UMEP-dev/UMEP#570] [Q&A]
Hi,

I´m using spatial thermal comfort to generate a PET map but I had this message:

Traceback (most recent call last):
File "C:\Users/VLADIMIR/AppData/Roaming/QGIS/QGIS3\profiles\default/python/plugins\processing_umep\postprocessor\spatialtc_algorithm.py", line 274, in processAlgorithm
yyyyTmrt = int(filepath_tmrt[-18:-14])
ValueError: invalid literal for int() with base 10: '_202'

Could anyone help me please?

Thanks
![pet](https://github.com/UMEP-dev/UMEP/assets/124792761/c0df5d71-8e3a-4225-b20c-5f8d597f52be)

**Comment by biglimp** [comment]
Hm, what version of QGIS and UMEP do you use?

**Reply by biglimp** [reply]
Actually, now when I investigate, this is a bug. I will transfer this to issues and fix.
